pkg/policy/api/rule.go

```diff
@@ -561,24 +593,35 @@ const (
 	KafkaMaxTopicLen = 255
 )
 
-// KafkaMaxTopicVal is the maximum value of supported API Keys in KafkaAPIKeyMap
-// KafkaReverseAPIKeyMap
+// KafkaMaxAPIKeySupported is apiKey supported to expand a generic "produce" / "consume"
+// role to a set of low level apiKeys.
+const (
+	KafkaMaxAPIKeySupported = 11
+)
+
+// KafkaInvalidAPIKey is the placeholder for a invalid apiKey value -1
```

Comment:
Please move the comment right above the constant so it works with godoc

Reply:
sure